TimeReport.py

```diff
@@ -76,6 +76,12 @@ def peopleTime(date):
 				enteredTime += hours
 			if enteredTime >= hoursForAcceptance:
 				marked = True
+			# Custom check for interns
+			if args.interns:
+                                internName = first + " " + last
+                                # Checks whether person is an intern and if they have worked yesterday
+                                if internArgs.get(internName) and dayOfWeek in internArgs.get(internName) and enteredTime > 0:
```

Comment:
Also as Caden mentioned try and think of another way of getting the day without using the value that is set further down